app/screens/Chat/ChatScreen.tsx

```diff
@@ -2,19 +2,26 @@ import { SafeAreaView } from 'react-native';
 import { useDispatch, useSelector } from 'react-redux';
 
 import { currentGroupSelector, groupsActions } from '@Stores/groups';
+import { getUserStatusByIdSelector, userIdSelector } from '@Stores/user';
 
 import { styles } from './ChatScreenStyles';
 import { Header } from './components/Header';
 import { ListChatsContainer } from './containers/ListChatsContainer';
 
 export const ChatScreen = () => {
   const currentGroup = useSelector(currentGroupSelector);
+  const userId = useSelector(userIdSelector);
   const dispatch = useDispatch();
 
   const handleClickGoBack = () => {
     dispatch(groupsActions.setCurrentGroupId(''));
   };
 
+  const otherMember = currentGroup?.members.filter((member) => member._id !== userId);
```

Comment:
Seem like this logic happen 2 times, please try to wrap both of them into 1.

Reply:
I don't think so, because for creating new hook to get this data, we need groupId and select them and not meaning for me